### PR TITLE
add kubectl.kubernetes.io/default-logs-container deprecated information

### PR DESCRIPTION
**Pull request by pacoxu**
Fixes https://github.com/kubernetes/enhancements/issues/2227

### CODE REVIEW

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -625,6 +625,17 @@ Example: `kubectl.kubernetes.io/default-container: "front-end-app"`
...
 The value of the annotation is the container name that is default for this Pod. For example, `kubectl logs` or `kubectl exec` without `-c` or `--container` flag will use this default container.
 
+### kubectl.kubernetes.io/default-logs-container (deprecated)
+
+Example: `kubectl.kubernetes.io/default-logs-container: "front-end-app"`
+
+The value of the annotation is the container name that is the default logging container for this Pod. For example, `kubectl logs` without `-c` or `--container` flag will use this default container.
+
+{{< note >}}
+Starting from Kubernetes 1.21, this annotation is deprecated and please use `kubectl.kubernetes.io/default-container` annotation instead.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:635]:
```suggestion
This annotation is deprecated. You should use the
[`kubectl.kubernetes.io/default-container`](#kubectl-kubernetes-io-default-container)
annotation instead.
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -625,6 +625,17 @@ Example: `kubectl.kubernetes.io/default-container: "front-end-app"`
...
+Kubernetes 1.25 and newer will ignore this annotation.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:636]:
```suggestion
Kubernetes versions 1.25 and newer ignore this annotation.
```

**Review by sftim — COMMENTED** [review]
/lgtm

I made suggestions; accepting those is optional.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 1 above)
Comment (pacoxu) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: Updated. Thanks for your review.

**Pull request merged**